packages/react/slider/src/collection.tsx

```diff
@@ -1,130 +0,0 @@
-import * as React from 'react';
```

Comment:
Nice to see all this go, maybe the package will even slim down a bit 😀 
Not sure…

Reply:
I wondered the same haha 🤞